src/lib.rs

```diff
@@ -157,7 +157,7 @@ fn has_internal_caps(word: &str) -> bool {
 }
 
 fn has_internal_slashes(word: &str) -> bool {
-    !word.is_empty() && word[1..].contains('/')
+    !word.is_empty() && word.chars().skip(1).collect::<String>().contains('/')
```

Comment:
This one is the source of the panic. There's no need to allocate a string though, we can iterate over the characters and check if one is a slash.

Something like:

```rust
fn has_internal_slashes(word: &str) -> bool {
    word.chars().skip(1).any(|chr| chr == '/')
}
```